src/components/pages/ImageClassification/ImageAnnotationModal/ImageAnnotationPopup/ImageAnnotationPopup.jsx

```
@@ -1,136 +1,140 @@
-    (annotation) => annotation.is_machine_created,
-  ).length
+                                  dataToReview,
+                                  setDataToReview,
+                                  pointId,
+                                  databaseSwitchboardInstance,
+                                  setIsDataUpdatedSinceLastSave,
+                                  resetZoom,
+                                  zoomToSelectedPoint,
+                                  selectNextUnconfirmedPoint,
+                              }) => {
+    const {t, i18n} = useTranslation()
```

ESLint

[error] 'i18n' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)